### PR TITLE
Docs to RTD.

### LINKED ISSUE #15: Move docs to RTD
I was trying to cross-reference this project and discovered I couldn't.

### PR DESCRIPTION
**Pull request by jamadden**
Viewable at https://zopeformlib.readthedocs.io/en/rtd-docs/

Fixes #15

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+
+# Example configuration for intersphinx: refer to the Python standard library.
+intersphinx_mapping = {
+    'http://docs.python.org/': None,
+    'http://zopebrowser.readthedocs.io/en/latest': None,
+    'http://zopecomponent.readthedocs.io/en/latest': None,
+    'http://zopeinterface.readthedocs.io/en/latest': None,
+    'http://zopepublisher.readthedocs.io/en/latest': None,
+    'http://zopeschema.readthedocs.io/en/latest': None,
```
Comment (mgedmin) [inline docs/conf.py]: Can we use https:// URLs here please?

File: src/zope/formlib/form.rst [hunk 2]
```diff
@@ -1975,7 +1980,7 @@ methods are accepted::
     Action: handle {'title': 'Submitted Title'}
 
 Prevent Cross-site Request Forgery (CSRF) attacks
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-------------------------------------------------
 
 See also: http://en.wikipedia.org/wiki/Cross-site_request_forgery.
```
Comment (mgedmin) [inline src/zope/formlib/form.rst]: I would also change this to https:// ;)

File: docs/conf.py (hunk 1 above)
Comment (jamadden) [reply docs/conf.py]: Sure. I always forget RTD also supports HTTS.

**Comment by jamadden** [comment]
Thanks!

**Pull request merged**